app/uk/gov/hmrc/traderservices/views/WaitingForFileVerificationView.scala.html

```diff
@@ -27,7 +27,7 @@
 @(successAction: Call, failureAction: Call, checkStatusAction: Call, backLink: Call)(implicit request: Request[_], messages: Messages)
 
 @govukLayout(
-  pageTitle = Some(messages("view.upload-file.waiting")),
+  pageTitle("view.upload-file.waiting"),
```

Comment:
It seems some of the other upload pages might need this change - error prefix doesn't prompt on /file-upload or /file-uploaded (both new & amend)

Reply:
there are no errors on /file-uploaded?